helm/cluster/templates/apps/apps.yaml

```diff
@@ -3,6 +3,9 @@
 {{- $app := dict }}
 {{- with $rootScope }}
 {{- $app = (tpl (.Files.Get $path) $) | fromYaml }}
+{{- $_ := set $ "appName" $app.appName }}
```

Comment:
Do you think it makes sense to remove [this line](https://github.com/giantswarm/cluster/pull/445/files#diff-e05e0c55f6c5d1cfe276b88160a71bce2e62fca1ed4fcf680c0d2813361668cfR39) then? Otherwise we're assigning it twice.